src/views/userOp/RecentUserOps.tsx

```diff
@@ -0,0 +1,615 @@
+    }, [hash]);
+
+    function getSymbol(network: string): string {
+        if (network == 'goerli') return 'ETH';
+        else if (network == 'mainnet') return 'ETH';
+        else if (network == 'mumbai') return 'MATIC';
+        else if (network == 'optimism-goerli') return 'ETH';
+        else if (network == 'matic') return 'MATIC';
+        else return 'ETH';
+    }
+    const getFee = (amount: number) => {
+        let fee: string = amount.toString();
+        if (amount > 10 ** 13) {
+            fee = (amount / 10 ** 18).toFixed(4).toString();
+        } else if (amount > 10 ** 6) {
+            fee = (amount / 10 ** 9).toFixed(4).toString();
+            // symbol = 'GWEI';
+        } else {
+            fee = amount.toString();
+            // symbol = 'WEI';
+        }
+        return fee;
+    };
+
+    return (
+        <div className="">
+            <Navbar searchbar />
+            <section className="py-10 px-3">
+                <div className="container">
+                    <div className="flex flex-row">
+                        <Link href="/" className="text-gray-500">
+                            <ArrowBackIcon
+                                style={{ height: '15px', width: '15px', marginRight: '20px', marginLeft: '10px', marginBottom: '3px' }}
+                            />
+                        </Link>
+                        <Breadcrumbs aria-label="breadcrumb" className="font-['Roboto']">
+                            <Link underline="hover" color="inherit" href="/">
+                                Home
+                            </Link>
+                            <Link underline="hover" color="text.primary" href="/recentUserOps" aria-current="page">
+                                Recent User Ops
+                            </Link>
+                            <Link underline="hover" color="text.primary" href={`/userOpHash?text=${hash}`} aria-current="page">
+                                {shortenString((hash as string) || '0x43fe1ef830cbc6447ca8a740963099fe7fb6b137ac7768aa9c8f5913aaf8f91b')}
+                            </Link>
+                        </Breadcrumbs>
+                    </div>
+                </div>
+            </section>
+            {useOpsData?.map((item) => {
+                return (
+                    <>
+                        <section className=" px-3">
+                            <div className="container  bg-white rounded shadow-200 px-[16px] py-[12px]">
+                                <h3 className="text-[15px]  leading-[28px] font-bold text-dark-600">User Operation Hash</h3>
+                                <div className="md:flex items-center gap-4 pt-[14px] pb-[2px]">
+                                    <div className="flex items-center gap-2">
+                                        <img src="/images/ethereum.svg" alt="" />
+                                        <span className="text-sm font-normal leading-5 text-dark-600">{item.network}</span>
```

Comment:
In the constants file , we have a network information object. Please pull the network name from there. https://github.com/jiffy-labs/jiffyscan-frontend/blob/master/src/components/common/constants.ts#L3